jobs/aws/eks-distro/release-1-18-postsubmits.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+postsubmits:
+  aws/eks-distro:
+  - name: release-1-18-postsubmit
+    always_run: false
+    run_if_changed: "release/1-18/RELEASE"
```

Comment:
Im not sure this is the right trigger actually.  When we create a new release in the repo we usually add this file from the get go.  I *think* the various builds that happen will look at it for tagging.  for this to be correct, wed have to make sure when we add a new version we exclude this RELEASE file.  Also, at this point 1.19 has the RELEASE file already set to 1. How will we trigger this, keeping the release 1?

Reply:
We would have to name this is `CURRENT_RELEASE` or `LATEST_RELEASE` or something. Unfortunately I don't think Prow supports triggers by GIT tags, otherwise that would solve our problem. But we are basically using this file as tags for us to be able to do it. Otherwise, we can have something like `release/1-18/v1-18-eks-1` and then for our Makefile target to cut a release based on the latest release file that exists, but that is a bit hacky too.

Reply:
Do you think, the trigger should be just "release/1-18"?

Reply:
Ideally, we'd probably be working on a branch and merge it and that file changing would trigger the release. That way we aren't breaking regular uses working on main

Reply:
The branch would have the new version like 2 for example.

Reply:
Im not sure its realistic to think that we would do everything for a given release of kubernetes on 1 branch.  For releases > 1, I can kinda see the point, the last thing we do is bump the release file and that triggers the build.  But for the first release of a given minor version, Im not sure this is best.